django-verdant/rca/templates/rca/includes/enquiry_form.html

```diff
@@ -0,0 +1,48 @@
+{% load wagtailcore_tags widget_tweaks %}
+
+{% if enquiry_page %}
+  <form action="{% pageurl enquiry_page %}" method="POST">
+  {% csrf_token %}
```

Comment:
As per discussion in Slack: we need to disable the csrf token validation for this form page or load this form using XMLHttpRequest.